enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/load_balancing/procedure/GetServersProcedureV1Test.java

```diff
@@ -365,6 +370,20 @@ private ClusterView run( GetServersProcedureV1 proc ) throws ProcedureException
         return ClusterView.parse( (List<Map<String,Object>>) rows[1] );
     }
 
+    public static Map<MemberId,ReadReplicaAddresses> addresses( int... ids )
```

Comment:
The local helper gives back `Map<MemberId,ReadReplicaAddresses>` rather than `ReadReplicaAddresses`.